models/POP/shell_scripts/cesm2_1/st_archive.sh

```diff
@@ -0,0 +1,400 @@
+#!/bin/sh
+#
+# This code is part of the CESM distribution,
+# So it is not protected by the DART copyright agreement.
```

Comment:
Is this different than the one under CESM? What is different? If this file exists in the CESM distrubution, I try to provide the full filename as a comment so people can see what is different.